randomwallpaper@iflow.space/settings.ui

```diff
@@ -1004,6 +1019,13 @@ You can also define a prefix that will be added to the image URL.</property>
             <property name="can-focus">1</property>
           </object>
         </child>
+        <child>
+          <object class="GtkCheckButton" id="wallhaven-allow-nsfw">
+            <property name="label" translatable="yes">NSFW</property>
```

Comment:
I think, the information that an API key is required would be a good addition to the name (i.e., "NSFW (API key required)")